### DISCUSSION TITLE
Select border not visible

### DISCUSSION THREAD

**Discussion by bourgeoisadeline** [lookfirst/mui-rff#951] [General]
**Describe the bug**
Hello,
I use the Select from mui-rff and we don't see the border anymore since I updated mui-rff package, We can see it only on hover/focus. I noticed the different between this select and the one from mui/core and it's on the fieldset for the select, we have a white border now.

**To Reproduce**
Use a Select or even an Texfield with any kind of variant from mui-rff.

https://user-images.githubusercontent.com/107409609/209938974-5b9dc034-50aa-4498-b9a7-be5d998b5caa.mov

**Expected behavior**
To see the border. Like on the select from mui/core.
![Capture d’écran 2022-12-29 à 11 30 37](https://user-images.githubusercontent.com/107409609/209938829-ab9c2f8d-5e01-4a25-bfc1-c01156cfef84.png)

![Capture d’écran 2022-12-30 à 10 59 35](https://user-images.githubusercontent.com/107409609/210057954-18466767-37ca-48dd-a594-e0e141e85b3f.png)

**Screenshots**
On the fieldset
![Capture d’écran 2022-12-29 à 11 28 06](https://user-images.githubusercontent.com/107409609/209938577-df78506c-f7f2-4cc3-afc3-349d4bf58484.png)

Same for Textfield, the first one is on hover/focus
![Capture d’écran 2022-12-30 à 10 31 43](https://user-images.githubusercontent.com/107409609/210055359-9fff9a9d-b52a-47d3-a783-0945252b10dc.png)
![Capture d’écran 2022-12-30 à 10 33 23](https://user-images.githubusercontent.com/107409609/210055472-79ab4e40-af45-4bbf-8d7b-51896ddbf7d6.png)

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**
I don't know if it's an issue only on my side, I am on "mui-rff": "^6.1.2".

**Comment by lookfirst** [comment]
@bourgeoisadeline Thanks for the report. I think this is more of a discussion than an issue, but if we do find this is a problem with mui-rff, I'll convert back to an issue.

Given that [the demo](https://lookfirst.github.io/mui-rff/) does not have this issue, my guess is that it is something in your theme or overrides. I would try to recreate things with the least amount of changes and then work from there.

**Reply by bourgeoisadeline** [reply]
@lookfirst Thank you I will see on my side if there's an overrides or something.

**Reply by bourgeoisadeline** [reply]
I think I found where it's coming from, on @mui/material there is a condition on the theme palette mode, not sure how it works though
<img width="865" alt="Capture d’écran 2023-01-03 à 11 09 44" src="https://user-images.githubusercontent.com/107409609/210337001-e232c1cd-76e4-4c75-aa13-17167dec9c49.png">

**Reply by bourgeoisadeline** [reply]
I had to add the mode on my Mui theme, by default I didn't have, now it works.
I'm sorry for the inconvenience 😞

**Comment by lookfirst** [comment]
No inconvenience. Glad you found the issue!

**Reply by bourgeoisadeline** [reply]
Thank you
